src/Common/Security/ScanVulnerabilitiesTable.tsx

```diff
@@ -60,17 +99,35 @@ export default function ScanVulnerabilitiesTable({ vulnerabilities, hidePolicy,
     return (
         <table className="security-tab__table">
             <tbody>
-                <tr className={`security-tab__table-header ${shouldStick ? 'dc__position-sticky bcn-0 dc__zi-4 dc__top-0' : ''}`}>
+                <tr
+                    className={`security-tab__table-header ${shouldStick ? 'dc__position-sticky bcn-0 dc__zi-4 dc__top-42' : ''}`}
```

Comment:
This is a breaking change it should stick to top_0, if needed any other variant can specify a class to define top distance